spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/config/LoadBalancerAutoConfiguration.java

```diff
@@ -50,10 +54,14 @@
 @ConditionalOnProperty(value = "spring.cloud.loadbalancer.enabled", havingValue = "true", matchIfMissing = true)
 public class LoadBalancerAutoConfiguration {
 
+	private static final String ZONE_SPLITTER_COMMA = ",";
+
 	@Bean
 	@ConditionalOnMissingBean
 	public LoadBalancerZoneConfig zoneConfig(Environment environment) {
-		return new LoadBalancerZoneConfig(environment.getProperty("spring.cloud.loadbalancer.zone"));
+		return new LoadBalancerZoneConfig(environment.getProperty("spring.cloud.loadbalancer.zone"),
+				List.of(Optional.ofNullable(environment.getProperty("spring.cloud.loadbalancer.secondary-zones"))
```

Comment:
Please don't use `Optional` as an if statement

Reply:
I removed Optional / StringUtil. Instead of this, I set default value of the property when any input exists.